devices/Sim7080/samples/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
```

Comment:
you can make this one a bit simpler. See for example https://github.com/nanoframework/nanoFramework.IoT.Device/blob/develop/devices/Adxl345/Properties/AssemblyInfo.cs

Reply:
Changed this following the example.

Reply:
Changed it following the example.